script/Arcueid/Day 7/07_00_ARC07_3E.txt

```diff
@@ -244,11 +242,13 @@ I remove my glasses.
 [sha:8140963fa807ccac93e0956153ffb69ba58cac36]{
 -- Page 290, Offset 17285.
 -- こめかみには軽い頭痛。
+<<<<<<< Updated upstream
 A light headache pulses in my temple.
+=======
 }
 [sha:052c2b4f9c429ba0bad540c9aac8ac0177f1ba34]{
 -- Page 290, Offset 17286.
--- その痛みと引き換えに、うっすらと、地面や壁に“線”が視える。
+-- その痛みと引き換えに、うっすらと、地面や壁に“線”uffが視える。
```

Comment:
Merge junk?

Reply:
Might just be some random junk I accidentally typed in while hovering. It's a bit of a bad habit of mine to random press keys while I'm hovering on text and thinking about what to type.